### ISSUE TITLE
inputs.disk with mount_points errors in 1.12.x

### ISSUE THREAD

**Issue by boombatower** [influxdata/telegraf#6388]
### Relevant telegraf.conf:
```toml
[[inputs.disk]]
  mount_points = [
    "/hostfs/vol/ssd/0_400G",
    "/hostfs/vol/hdd/0_3T",
    "/hostfs/vol/hdd/2_1T",
    "/hostfs/vol/hdd/3_4T",
    "/hostfs/vol/hdd/4_4T",
  ]
```

### System info:

- works: telegraf 1.11.5 (1.11.x all work) via official docker
- fails: telegraf 1.12.1 (1.12.0 also fails) via official docker

### Steps to reproduce:

1. start telegraf with above config

### Expected behavior:

works as before

### Actual behavior:

```
2019-09-12T19:21:40Z E! [inputs.disk] Error in plugin: error getting disk usage info: lstat /dev/mapper/vol_ssd_0_400G_crypt: no such file or directory
```

**Comment by xkfen** [comment]
> I copied my config and only ran disk and output to stdout. I mounted the binaries from `/usr/bin` of tarball via: `-v /root/telegraf:/usr/bin/telegraf:ro` and ran latest (1.12.1) docker image.
> 
> With the binary override mount proper output was generated:
> 
> ```
> disk,device=mapper/vol_hdd_2_1T_crypt,fstype=ext4,mode=ro,path=/vol/hdd/2_1T used=18886762496i,used_percent=2.0234354188704478,inodes_total=61054976i,inodes_free=61010063i,inodes_used=44913i,total=983427665920i,free=914514042880i 1568346480000000000
> ```
> 
> Without the binary override mount it failed like normal:
> 
> ```
> [inputs.disk] Error in plugin: error getting disk usage info: lstat /dev/mapper: no such file or directory
> ```
> 
> As such I can confirm your build works.

hi, i had the same problem:
     [inputs.disk] Error in plugin: error getting disk usage info: lstat /dev/mapper: no such file or directory.
     how did you solve this problem? thanks

**Comment by andyshutak** [comment]
Yea im also having that problem on the latest docker container

**Comment by cookandy** [comment]
Same! Can we get a new docker release with the fix please?

**Comment by glinton** [comment]
I believe this will be going out in the release on Tuesday. If you are unable to wait that long, 
consider using a [nightly build](https://github.com/influxdata/telegraf#nightly-builds) which should also work.

**Comment by danielnelson** [comment]
Closed in #6389
